addons/point_of_sale/views/report_saledetails.xml

```diff
@@ -4,86 +4,417 @@
+                                <td style="background-color: #b1b1b1;"/>
+                                <td style="background-color: #b1b1b1;">
+                                    <strong><t t-esc="products_info['qty']"/></strong>
+                                </td>
+                                <td style="background-color: #b1b1b1;">
+                                    <t t-if="currency['position']">
+                                        <strong><t t-esc='products_info["total"]' t-options="{'widget': 'float', 'precision': currency['precision']}"/><t t-esc='currency["symbol"]' /></strong>
+                                    </t>
+                                    <t t-else="" >
+                                        <strong><t t-esc='currency["symbol"]' /><t t-esc='products_info["total"]' t-options="{'widget': 'float', 'precision': currency['precision']}"/></strong>
+                                    </t>   
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
                 </t>
+
+                <t t-if="taxes" id="taxe_sales">
+                    <h3>Taxes on sales</h3>
+                    <table  class="table table-sm">
+                        <thead><tr>
+                            <th>Name</th>
+                            <th>Tax Amount</th>
+                            <th>Base Amount</th>
+                        </tr></thead>
+                        <tbody>
+                        <tr t-foreach='taxes' t-as='tax'>
+                            <td><t t-esc="tax['name']" /></td>
+                            <td>
+                                <t t-if="currency['position']">
+                                    <t t-esc="tax['tax_amount']" t-options="{'widget': 'float', 'precision': currency['precision']}"/><t t-esc='currency["symbol"]' />
+                                </t>
+                                <t t-else="" >
+                                    <t t-esc='currency["symbol"]' /><t t-esc="tax['tax_amount']" t-options="{'widget': 'float', 'precision': currency['precision']}"/>
+                                </t>
+
+                            </td>
+                            <td>
+                                <t t-if="currency['position']">
+                                    <t t-esc="tax['base_amount']" t-options="{'widget': 'float', 'precision': currency['precision']}"/><t t-esc='currency["symbol"]' />
+                                </t>
+                                <t t-else="" >
+                                    <t t-esc='currency["symbol"]' /><t t-esc="tax['base_amount']" t-options="{'widget': 'float', 'precision': currency['precision']}"/>
+                                </t>
+                            </td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </t>
```

Comment:
Maybe we could add the total of both tax_amount and base_amount at the bottom of the table